src/main/java/org/resitrack/controller/PersonController.java

```diff
@@ -51,13 +94,76 @@ public void searchPeopleByName() {
         }
     }
 
+    /**
+     * This function is used to delete one or many people
+     *
+     * @param name
+     * @return true if delete process success and vice versa
+     */
+    public boolean deletePeopleByName(String name) {
+        List<Person> peopleByName = findPersonByName(name);
+        if (peopleByName.size() < 0) {
```

Comment:
```suggestion
        if (peopleByName.size() == 0) {
```
the logic here should check for == 0

Reply:
Đã update